Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -53,6 +53,12 @@ struct TestCommand: ParsableCommand {
     )
     var skipUITests: Bool = false
 
+    @Flag(
+        name: .long,
+        help: "When passed, it shows the testable schemes."
+    )
+    var showSchemes: Bool = false
```

Comment:
What about naming this `listSchemes`? It might feel more natural for people using `xcodebuild` where to get project's schemes, you run `xcodebuild -list`.

Reply:
Agree as well, I wrote yesterday both options and I felt `--list-schemes` was way more natural 👍